src/mixins/add-edit-org-person-mixin.ts

```diff
@@ -199,46 +200,17 @@ export default class AddEditOrgPersonMixin extends Vue {
   }
 
   /** Whether the Incorporator role should be disabled. */
-  // *** TODO: test this (should be disabled if it's the only role displayed)
   get disableIncorporatorRole (): boolean {
     // disable this role if it's the only role displayed
     return (!this.showCompletingPartyRole && !this.showDirectorRole && this.showIncorporatorRole)
   }
 
   /** Whether the Director role should be disabled. */
-  // *** TODO: test this (should be disabled if it's the only role displayed)
   get disableDirectorRole (): boolean {
-    // always enable this role for restoration filings
-    if (this.isFullRestorationFiling || this.isLimitedRestorationFiling) return false
```

Comment:
This getter is no longer used for restoration filings.

This also means the TODOs on lines 202 and 209 are no longer needed (since it's the same code as before).